### ISSUE TITLE
Port 1 not allowed

### ISSUE THREAD

**Issue by Wooble** [liamcottle/atlas-for-rust#1]
I'm getting the message "The port 1 is not allowed." immediately after pairing for the first time after installing Atlas 1.0.2

![image](https://user-images.githubusercontent.com/223942/121888624-36de1300-cce6-11eb-9605-68ee2f4261ac.png)

**Comment by liamcottle** [comment]
This looks unusal. The port shouldn't be `1`. Either something went wrong while decoding the fcm notification when you tried to pair, or the server is configured incorrectly.

Are you able to connect to this specific server in the official Rust+ app?

Also, it may help if you can run the `fcm-listen` command from the readme, and see what data is being sent when you try to pair.

**Comment by Wooble** [comment]
I was able to connect to this server with Rust+ at the same time I was having this problem. However, now I removed it from both Rust+ and Atlas, and tried to reconnect it, and neither of them seems to be getting a notification to connect at all. I'm able to connect Atlas to a different server just fine, so clearly something weird is and was going on with this specific server.

I'm not finding anything about fcm-listen in the README, although I suspect it wouldn't help much now if Rust+ also isn't getting any data to try to pair...

In any event, doesn't look like a problem with Atlas specifically, even if Rust+ somehow figured out what to do with the incorrect request, and I can't repro anymore so... closing.
